shared/constants/platform.desktop.js

```diff
@@ -27,196 +23,79 @@ if (__DEV__ && !__STORYBOOK__) {
   console.log(`Run mode: ${runMode}`)
 }
 
-const envedPathLinux = {
-  devel: 'keybase.devel',
-  prod: 'keybase',
-  staging: 'keybase.staging',
-}
-
-const envedPathOSX = {
-  devel: 'KeybaseDevel',
-  prod: 'Keybase',
-  staging: 'KeybaseStaging',
-}
-
-const envedPathWin32 = {
-  devel: 'KeybaseDevel',
-  prod: 'Keybase',
-  staging: 'KeybaseStaging',
-}
-
 const socketName = 'keybased.sock'
 
-function win32SocketDialPath(): string {
-  let appdata = process.env['LOCALAPPDATA'] || ''
-  // Remove leading drive letter e.g. C:
-  if (/^[a-zA-Z]:/.test(appdata)) {
-    appdata = appdata.slice(2)
-  }
-  // Handle runModes, prod has no extension.
-  let extension = ''
-  if (runMode !== 'prod') {
-    extension = runMode.charAt(0).toUpperCase() + runMode.substr(1)
-  }
-  let dir = `\\\\.\\pipe\\kbservice${appdata}\\Keybase${extension}`
-  return path.join(dir, socketName)
-}
-
-function linuxSocketDialPath(): string {
+const getLinuxPaths = () => {
   // If XDG_RUNTIME_DIR is defined use that, else use $HOME/.config.
   const homeConfigDir = path.join(homeEnv, '.config')
   const runtimeDir = process.env['XDG_RUNTIME_DIR'] || ''
-
   const cacheDir = runtimeDir || homeConfigDir
-  const suffix = runMode === 'prod' ? '' : `.${runMode}`
+  const appName = `keybase${runMode === 'prod' ? '' : `.${runMode}`}`
 
   if (!runtimeDir && !homeEnv) {
     console.warn(
       "You don't have $HOME or $XDG_RUNTIME_DIR defined, so we can't find the Keybase service path."
     )
   }
 
-  return path.join(cacheDir, `keybase${suffix}`, socketName)
-}
+  const logDir = `${process.env['XDG_CACHE_HOME'] || `${homeEnv}/.cache`}/${appName}/`
 
-const darwinCacheRoot = `${homeEnv}/Library/Caches/${envedPathOSX[runMode]}/`
-const darwinSandboxCacheRoot = `${homeEnv}/Library/Group Containers/keybase/Library/Caches/${
-  envedPathOSX[runMode]
-}/`
-const darwinSandboxSocketPath = path.join(darwinSandboxCacheRoot, socketName)
-
-function findSocketDialPath(): string {
-  switch (process.platform) {
-    case 'darwin':
-      return darwinSandboxSocketPath
-    case 'linux':
-      return linuxSocketDialPath()
-    case 'win32':
-      return win32SocketDialPath()
-  }
-  throw new Error(`Unknown platform ${process.platform}`)
-}
-
-function findDataRoot(): string {
-  switch (process.platform) {
-    case 'darwin':
-      return `${homeEnv}/Library/Application Support/${envedPathOSX[runMode]}/`
-    case 'linux':
-      const linuxDefaultRoot = `${homeEnv}/.local/share`
-      return `${process.env['XDG_DATA_HOME'] || linuxDefaultRoot}/${envedPathLinux[runMode]}/`
-    case 'win32':
-      return `${process.env['LOCALAPPDATA'] || ''}\\Keybase\\`
```

Comment:
for every other path on every platform we always use Keybase${suffix} except this, which seems like a bug, so in the new code it does this also